content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -7,6 +7,7 @@ weight: 100
 
 This section contains technical details about programming in the Amazon Shader Language (AZSL) and compiling with the Amazon Shader Language Compiler (AZSLc). In the Atom Renderer, shaders are written in AZSL and are compiled by AZSLc. 
 
+<!-- Zero length topics have been set to draft
```

Comment:
Are the "placeholder" pages in `.../shaders/azsl/` supposed to be set to draft? I don't see those changes here, and the pages still appear in the navigation.

Reply:
Those were in a prior PR that has since been merged. https://github.com/o3de/o3de.org/pull/798/
This PR is fixing broken link issues called out in the previous PR, and did not include 798 when it was submitted.